ckan-master/base/Dockerfile

```diff
@@ -40,8 +40,6 @@ RUN apk add --no-cache git \
         uwsgi-http \
```

Comment:
For master (CKAN 2.11) we won't have alpine based images any more so let's remove this file and name the Py3 based ones just `Dockerfile`, also the `start_ckan.*.sh` files. So we only have `-py3` files in 2.10 where alpine and py3 images coeexist.
WDYT?

Reply:
@amercader OK, Iv'e made the changes to master, ready for review now